logger/main.go

```diff
@@ -32,6 +32,9 @@ func init() {
 	flag.IntVar(&logPort, "log-port", 514, "bind port for the logger")
 	flag.StringVar(&drainURI, "drain-uri", "", "default drainURI, once set in etcd, this has no effect.")
 	flag.StringVar(&syslogd.LogRoot, "log-root", "/data/logs", "log path to store logs")
+	flag.StringVar(&syslogd.HandlerType, "handler-type", "standard", "Syslog handler type, 'standard' or 'ringbuffer'")
+	flag.IntVar(&syslogd.RingBufferSize, "ring-buffer-size", 1000, "Size of log buffer for each application")
```

Comment:
Wondering if there's a cleaner separation of concerns that can be achieved here.  `RingBufferSize ` seems like it should be something exposed by the `ringbuffer` package-- which doesn't exist yet, but should-- instead of just being glommed onto `syslogd`.